app/views/shared/forms/_body.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
The indentation here is out of alignment

Reply:
👍